crates/ruma-events/src/media_preview_config.rs

```diff
@@ -0,0 +1,132 @@
+//! Types for the [`m.media_preview_config`] event.
+//!
+//! [`m.media_preview_config`]: https://github.com/matrix-org/matrix-spec-proposals/pull/4278
+
+use ruma_macros::StringEnum;
+use serde::{Deserialize, Serialize};
+
+use crate::{macros::EventContent, PrivOwnedStr};
+
+/// The content of an `io.element.msc4278.media_preview_config` event,
+/// the unstable version of `m.media_preview_config` in global account data.
+#[derive(Clone, Debug, Deserialize, Serialize, EventContent)]
+#[cfg_attr(not(ruma_unstable_exhaustive_types), non_exhaustive)]
+#[ruma_event(type = "io.element.msc4278.media_preview_config", kind = GlobalAccountData)]
+pub struct UnstableMediaPreviewConfigEventContent {
+    /// The media previews configuration.
+    #[serde(rename = "media_previews")]
```

Comment:
I don't know why I didn't check this before, but we don't need to use `rename` if the field name doesn't change. Same below.